setup.py

```diff
@@ -96,6 +96,7 @@
             "types-dateparser",
             "types-python-dateutil",
             "types-pytz",
+            "types-PyYAML",
```

Comment:
This is not needed. We use ruamel in SQLmesh

Reply:
Ok, I guess it was only needed because of the import